src/main/java/lotto/domain/PurchaseAmount.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain;
+
+import static lotto.util.ErrorMessage.*;
+
+import java.util.HashMap;
+
+public class PurchaseAmount {
+
+    private static final int REMAINDER = 0;
+    private static final int AMOUNT_UNIT = 1000;
+    private static final int INPUT_AMOUNT_MIN_LIMIT = 1000;
+    private static final int INPUT_AMOUNT_MAX_LIMIT = 100000;
```

Comment:
숫자 가독성을 위해서 _(언더바) 를 쓰시는 것 추천드립니다! 
```suggestion
    private static final int INPUT_AMOUNT_MAX_LIMIT = 100_000;
```